bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java

```diff
@@ -160,13 +160,13 @@ public void handleCommand(ChannelUID channelUID, Command receivedCommand) {
                     }
                     String cmd = action.getCommand();
                     CommandParameterType paramType = action.getparameterType();
-                    if (command instanceof QuantityType) {
+                    if (command instanceof QuantityType type) {
```

Comment:
same here. In my mind it obscures the code rather than make it more readable

Reply:
In my humble opinion it does not justify breaking the code compatibility.
Suggest to keep those out of the update for miio binding

Reply:
resolving conversation, as miio is no longer part of this PR